org.gnome.dspy.json

```diff
@@ -0,0 +1,85 @@
+{
+    "app-id": "org.gnome.dspy",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "40",
+    "sdk": "org.gnome.Sdk",
+    "command": "d-spy",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=session-bus",
+        "--socket=system-bus"
```

Comment:
I'm gonna assume that given what dspy is this is essential :-)

Reply:
yup yup